app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command_line: str) -> None:
+    _, source_name, destination_name = command_line.split(" ")
+
+    if source_name != destination_name:
```

Comment:
It is usually considered best practice not to put function's main logic into an if condition. I'd suggest to rewrite the check so that the function exits `if source_name == destination_name` and proceeds to execute otherwise.